### PR TITLE
Improving PlotlyViaWindowModule docs

### LINKED ISSUE #36: Improve docs for PlotlyViaWindowModule
@andrefarzat  PlotlyViaWindowModule example is a bit hard to understand for the people who are new to the angular community. Can you give a full example in the readme file?

_Originally posted by @alwaysprep in https://github.com/plotly/angular-plotly.js/issues/5#issuecomment-457079057_

### PR DESCRIPTION
**Pull request by andrefarzat**
@alwaysprep do you mind reviewing this? 
Closes #36

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -172,7 +174,33 @@ export class AppModule { }
 
 ### Plotly Via Window Module
 
-If you want to use a [different precompiled bundle](https://github.com/plotly/plotly.js/blob/master/dist/README.md#partial-bundles) or if you wish to [assemble you own customized bundle](https://github.com/plotly/plotly.js#modules), you can use `PlotlyViaWindowModule` to force the use of `window.Plotly` object. You can add the script via tag `<script>` direct on your code, or add it as a [global script on angular.json](https://github.com/angular/angular-cli/wiki/stories-global-scripts#global-scripts).
+`plotly.js` can be added as a [global script on angular.json](https://github.com/angular/angular-cli/wiki/stories-global-scripts#global-scripts) to avoid it beingb bundled into the final project's code. To make this happen, you must first add `plotly.js` path into `angular.json` file as shown below:
```
Comment (ryanjackman) [inline README.md]: typo: beingb

**Comment by ryanjackman** [comment]
I wasn't the person who requested this but I found these changes helpful. I was able to switch to a smaller bundle of plotly.js.

LGTM

**Comment by alwaysprep** [comment]
Thanks @andrefarzat it is more clear now. As @ryanjackman point out there is a typo there, other than that it is great.
I have one more question, as far as I understand tree shaking does not work on this situation since plotly.js is put on vendor.js no matter what. Am I wrong?

**Comment by andrefarzat** [comment]
@alwaysprep exactly. As far as I understood, the tree-sharking working only on the code which passes through the ivy compiler. Although, you can still create your own plotly.js bundle to optimize the size

**Pull request merged**